### PR TITLE
Adapt doc

### PR DESCRIPTION
**Pull request by Yozhgoor**
Want to make so improvement to the doc:

- card:
   - change the elevation menu to a select

- Switch:
   - make something
- Menu:
   - make something

### CODE REVIEW

File: yewprint/src/switch.rs [hunk 1]
```diff
@@ -37,10 +44,16 @@ impl Component for Switch {
...
     fn view(&self) -> Html {
         html! {
-            <label class="bp3-control bp3-switch">
+            <label class=(
+                "bp3-control bp3-switch",
+                self.props.disabled.map_some("bp3-disabled"),
+                self.props.inline.map_some("bp3-inline"),
+                self.props.large.map_some("bp3-large"),
+                )
+            >
```
Comment (cecton) [inline yewprint/src/switch.rs:53]:
```suggestion
            <label
                class=(
                    "bp3-control bp3-switch",
                    self.props.disabled.map_some("bp3-disabled"),
                    self.props.inline.map_some("bp3-inline"),
                    self.props.large.map_some("bp3-large"),
                )
            >
```

File: yewprint/src/switch.rs [hunk 2]
```diff
@@ -37,10 +44,16 @@ impl Component for Switch {
...
             <input
                 type="checkbox"
-                checked={self.props.checked}
+                checked={self.props.checked},
```
Comment (cecton) [inline yewprint/src/switch.rs:56]:
```suggestion
                checked={self.props.checked}
```

**Comment by Yozhgoor** [comment]
i have some issues with the menu, i think we can already merge that and i will concentrate on the menu later

File: yewprint-doc/src/switch/example.rs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+            true
+        } else {
+            false
+        }
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            <div>
+                <H5>{"Privacy settings"}</H5>
```
Comment (cecton) [inline yewprint-doc/src/switch/example.rs:39]: It's a label in blueprint's doc: https://github.com/palantir/blueprint/blob/develop/packages/docs-app/src/examples/core-examples/switchExample.tsx#L28

But overall you don't need to replicate blueprint's example exactly. You can also invent fun things.

File: yewprint-doc/src/card/mod.rs [hunk 4]
```diff
@@ -72,46 +72,22 @@ crate::build_example_prop_component! {
...
+                                (Some(Elevation::Level1), "Level 1".to_string()),
+                                (Some(Elevation::Level2), "Level 2".to_string()),
+                                (Some(Elevation::Level3), "Level 3".to_string()),
+                                (Some(Elevation::Level4), "Level 4".to_string()),
+                            ]}
+                            onchange=self.update_props(|props, elevation| ExampleProps {
+                                elevation,
+                                ..props
+                            })
+                        />
```
Comment (cecton) [inline yewprint-doc/src/card/mod.rs:90]: It's not an Option if it's always Some :bulb: 

<insert-meme-with-smart-guy-here>

File: yewprint/src/menu.rs [hunk 5]
```diff
@@ -149,3 +149,32 @@ impl Component for MenuItem {
...
+        true
+    }
+
+    fn view(&self) -> Html {
+        html! {
+            <span
+                class=(
+                    "bp3-menu-divider",
+                )
+            />
```
Comment (cecton) [inline yewprint/src/menu.rs:177]:
```suggestion
            <span class="bp3-menu-divider" />
```

**Pull request merged**